### DISCUSSION TITLE
Synocommunity package broken?

### DISCUSSION THREAD

**Discussion by zaadstra** [sabnzbd/sabnzbd#2656] [General]
First: it this the proper place to drop questions on Synocommunity sabnzbd? I don't see a discussion place there.  Please point me in the right direction if there is a better place for this.

The NAS offered a new package update, so I updated. Then the message is "cannot start package service".  Rebooted, no luck. Uninstall/reinstall, same message. Then grabbed previous package sabnzbd.v63.f25556[qoriq].spk and this worked. Unfortunately it did not find the old config and history. Luckily I had an export backup from a week ago. This is the first time that an update breaks the installation.

Yesterday the update was refused to work because of Python 311.  Today an update for Python 311 appeared. But after install and then update sabnzbd, the thing broke.

DSM 6.2,  Syno Qoriq architecture. Failing package 4.0.3-64.

Note, the update on a more recent NAS worked fine, this is with a Celeron J3355 (not sure with arch they call it).

**Comment by Safihre** [comment]
I think you are having the same problem as here:
https://github.com/SynoCommunity/spksrc/issues/5841

**Reply by zaadstra** [reply]
Looks like it although my issue started from -64 and not -63.  Thanks for the heads up, let's close this one.

I saw the spksrc tree but did not realise that packages like sab are below that ;-)
